shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/api/impl/RuleAlteredJobAPIImpl.java

```diff
@@ -80,6 +80,12 @@
     private static final Map<String, DataConsistencyCheckAlgorithm> DATA_CONSISTENCY_CHECK_ALGORITHM_MAP = new TreeMap<>(
             SingletonSPIRegistry.getTypedSingletonInstancesMap(DataConsistencyCheckAlgorithm.class));
     
+    private static final JobProgressYamlSwapper JOB_PROGRESS_YAML_SWAPPER = new JobProgressYamlSwapper();
+    
+    private static final String SCALING_OFFSET_PLACEHOLDER = "%s/%s/offset";
+    
+    private static final String SCALING_OFFSET_SHARDING_ITEM_PLACEHOLDER = "%s/%s/offset/%d";
```

Comment:
Could we put scaling offset path in `GovernanceRepositoryAPIImpl`, make sure they're in the same place. And also related logic code.

New method could be added in `GovernanceRepositoryAPI`.